docs/plugins/repository-source-only.asciidoc

```diff
@@ -0,0 +1,39 @@
+[[repository-src-only]]
```

Comment:
@clintongormley @debadair I'd love to get input where this should be linked from and where it should be located. at this point it's stand-alone.